src/reflection/FStar.Reflection.Basic.fs

```diff
@@ -537,6 +537,12 @@ let inspect_sigelt (se : sigelt) : sigelt_view =
         in
         Sg_Inductive (nm, us, param_bs, ty, List.map inspect_ctor c_lids)
 
+    | Sig_declare_typ (lid, us, ty) ->
+        let nm = Ident.path_of_lid lid in
+        let s, us = SS.univ_var_opening us in
+        let ty = SS.subst s ty in
```

Comment:
This could be simplified to `let us, ty = SS.open_univ_vars us ty`.

Reply:
Thanks! I pushed an updated version